File: class-creation.rb
require 'pry'

class Airplane

    # attr_reader :color, :type
    # attr_writer :color, :type

    attr_accessor :color, :type

    def initialize(type, color="blue")
        @type = type
        @color = color
    end

    # def color=(color) # setter (writers)
    #     @color = color
    # end

    # def color # getter (readers)
    #     @color
    # end

    #getter
    # def type
    #     @type
    # end

    #setter
    # def type=(type)
    #     @type=type
    # end

    def fly
        puts "I am flying away!!!"
    end

    def fly_to(location, mph)
        puts "I am flying to #{location} at #{mph} miles per hour"
    end

end

airplane = Airplane.new("747", "green")

binding.pry

File: class-creation.js
class Airplane {
    constructor(color, type) {
        this.color = color;
        this.type = type
    }

    fly() {
        console.log("The airplane is flying away!!!!")
    }
}

const airplane = new Airplane("blue", "747")

console.log(airplane)
console.log(airplane.fly())